README.md

```diff
@@ -1 +1,55 @@
-# Hello_word
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello_word
+
+> One paragraph statement about the project.
+
+![screenshot](./readme.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html and CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Clone git repo
+
+### Run the index.htm
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@m05e5](https://github.com/m05e5)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
[OPTIONAL] This should be a link to the issues page.

Reply:
thanks